pkg/config/config.go

```diff
@@ -264,167 +264,168 @@ func DefaultAPIConfig() APIConfig {
-						},
+					1: {
+						MinFPS: 5.0,
 					},
-					FrameTracker: map[int32]StreamTrackerFrameConfig{
-						0: {
-							MinFPS: 0.5,
-						},
-						1: {
-							MinFPS: 0.5,
-						},
-						2: {
-							MinFPS: 0.5,
-						},
+					2: {
+						MinFPS: 5.0,
 					},
 				},
 			},
-		},
-		Redis: redisLiveKit.RedisConfig{},
-		Room: RoomConfig{
-			AutoCreate: true,
-			EnabledCodecs: []CodecSpec{
-				{Mime: webrtc.MimeTypeOpus},
-				{Mime: "audio/red"},
-				{Mime: webrtc.MimeTypeVP8},
-				{Mime: webrtc.MimeTypeH264},
-				// {Mime: webrtc.MimeTypeAV1},
-				// {Mime: webrtc.MimeTypeVP9},
+			Screenshare: StreamTrackerConfig{
+				StreamTrackerType: StreamTrackerTypePacket,
+				BitrateReportInterval: map[int32]time.Duration{
+					0: 4 * time.Second,
+					1: 4 * time.Second,
+					2: 4 * time.Second,
+				},
+				PacketTracker: map[int32]StreamTrackerPacketConfig{
+					0: {
+						SamplesRequired: 1,
+						CyclesRequired:  1,
+						CycleDuration:   2 * time.Second,
+					},
+					1: {
+						SamplesRequired: 1,
+						CyclesRequired:  1,
+						CycleDuration:   2 * time.Second,
+					},
+					2: {
+						SamplesRequired: 1,
+						CyclesRequired:  1,
+						CycleDuration:   2 * time.Second,
+					},
+				},
+				FrameTracker: map[int32]StreamTrackerFrameConfig{
+					0: {
+						MinFPS: 0.5,
+					},
+					1: {
+						MinFPS: 0.5,
+					},
+					2: {
+						MinFPS: 0.5,
+					},
+				},
 			},
-			EmptyTimeout: 5 * 60,
-		},
-		Logging: LoggingConfig{
-			PionLevel: "error",
 		},
-		TURN: TURNConfig{
-			Enabled: false,
+	},
+	Redis: redisLiveKit.RedisConfig{},
+	Room: RoomConfig{
+		AutoCreate: true,
+		EnabledCodecs: []CodecSpec{
+			{Mime: webrtc.MimeTypeOpus},
+			{Mime: "audio/red"},
+			{Mime: webrtc.MimeTypeVP8},
+			{Mime: webrtc.MimeTypeH264},
+			// {Mime: webrtc.MimeTypeAV1},
+			// {Mime: webrtc.MimeTypeVP9},
 		},
-		NodeSelector: NodeSelectorConfig{
-			Kind:         "any",
-			SortBy:       "random",
-			SysloadLimit: 0.9,
-			CPULoadLimit: 0.9,
-		},
-		SignalRelay: SignalRelayConfig{
-			Enabled:          false,
-			RetryTimeout:     7500 * time.Millisecond,
-			MinRetryInterval: 500 * time.Millisecond,
-			MaxRetryInterval: 4 * time.Second,
-			StreamBufferSize: 1000,
-		},
-		Keys: map[string]string{},
-	}
+		EmptyTimeout: 5 * 60,
+	},
+	Logging: LoggingConfig{
+		PionLevel: "error",
+	},
+	TURN: TURNConfig{
+		Enabled: false,
+	},
+	NodeSelector: NodeSelectorConfig{
+		Kind:         "any",
+		SortBy:       "random",
+		SysloadLimit: 0.9,
+		CPULoadLimit: 0.9,
+	},
+	SignalRelay: SignalRelayConfig{
+		Enabled:          false,
+		RetryTimeout:     7500 * time.Millisecond,
+		MinRetryInterval: 500 * time.Millisecond,
+		MaxRetryInterval: 4 * time.Second,
+		StreamBufferSize: 1000,
+	},
+	Keys: map[string]string{},
+}
 
+func NewConfig(confString string, strictMode bool, c *cli.Context, baseFlags []cli.Flag) (*Config, error) {
+	// start with defaults
+	conf := DefaultConfig
```

Comment:
you'll probably also need to use `decoder.Decode` with a pointer to `conf`

Reply:
Changed in https://github.com/livekit/livekit/pull/1848/commits/c74b74753ba6f16a0bf2e0c91ae1f20a0722f0cc. Test passes now.